components/auto-complete/autocomplete.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { Directionality } from '@angular/cdk/bidi';
+import { Direction, Directionality } from '@angular/cdk/bidi';
```

Comment:
The changes to this file don't appear to be part of this PR, can you retract it?

Reply:
I knew i wrote test fot auto-complete once before :))
done